src/main/java/icons/SonarLintIcons.java

```diff
@@ -33,6 +33,7 @@ public class SonarLintIcons {
   public static final Icon ICON_SONARCLOUD_16 = IconLoader.getIcon("/images/sonarcloud-16.png");
 
   public static final Icon SONARLINT_TOOLWINDOW = IconLoader.getIcon("/images/sonarlintToolWindow.svg");
+  public static final Icon SONARLINT_TOOLWINDOW_EMPTY = IconLoader.getIcon("/images/sonarlintToolWindowEmpty.svg");
```

Comment:
This method is deprecated, I will change it after merging the PR about deprecations and rebasing on master